ami-update.sh

```diff
@@ -184,6 +184,7 @@ fi
 
 # Update kobo-docker, kobo-install with kobo-ec2 existing scripts
 echo-with-date "Updating KoBoToolbox on AMI..."
+$SSH "cd $KOBO_INSTALL_DIR; git fetch --tags"
```

Comment:
What about adding this command in the `start_env.bash` script of `kobo-ec2` repo instead?